### PR TITLE
[7.52.x] [DROOLS-6266] Fix failing PMML tests on Windows

### PR DESCRIPTION
**Pull request by jiripetrlik**

### CODE REVIEW

File: kie-pmml-trusty/kie-pmml-models/kie-pmml-models-drools/kie-pmml-models-drools-common/src/test/java/org/kie/pmml/models/drools/provider/DroolsModelProviderTest.java [hunk 1]
```diff
@@ -152,7 +152,12 @@ public void getKiePMMLModelWithSourcesWithKnowledgeBuilder() {
         PackageDescr packageDescr = knowledgeBuilder.getPackageDescrs(PACKAGE_NAME).get(0);
         commonVerifyPackageDescr(packageDescr, PACKAGE_NAME);
         assertNotNull(retrieved);
-        final String rootPath = PACKAGE_NAME + ".";
+        String rootPath;
```
Comment (gitgabrio) [inline kie-pmml-trusty/kie-pmml-models/kie-pmml-models-drools/kie-pmml-models-drools-common/src/test/java/org/kie/pmml/models/drools/provider/DroolsModelProviderTest.java]: @jiripetrlik 
Same as https://github.com/kiegroup/drools/pull/3520#discussion_r609656512

File: kie-pmml-trusty/kie-pmml-models/kie-pmml-models-drools/kie-pmml-models-drools-common/src/test/java/org/kie/pmml/models/drools/provider/DroolsModelProviderTest.java [hunk 2]
```diff
@@ -215,7 +220,12 @@ public void getRulesSourceMap() {
         PackageDescr packageDescr = knowledgeBuilder.getPackageDescrs(PACKAGE_NAME).get(0);
         final Map<String, String> retrieved = droolsModelProvider.getRulesSourceMap(packageDescr);
         assertNotNull(retrieved);
-        final String rootPath = PACKAGE_NAME + ".";
+        String rootPath;
+        if (System.getProperty("os.name").startsWith("Windows")) {
```
Comment (gitgabrio) [inline kie-pmml-trusty/kie-pmml-models/kie-pmml-models-drools/kie-pmml-models-drools-common/src/test/java/org/kie/pmml/models/drools/provider/DroolsModelProviderTest.java]: @jiripetrlik 
Same as https://github.com/kiegroup/drools/pull/3520#discussion_r609656512

**Review by gitgabrio — COMMENTED** [review]
Hi @jiripetrlik 
Thanks for the PR! I have the same comments as the twin one

**Review by gitgabrio — APPROVED** [review]
Thanks!

**Review by danielezonca — APPROVED** [review]

**Pull request merged**